mettle/worker/worker.go

```diff
@@ -1145,3 +1147,29 @@ func resultsCachePriority(skipCacheLookup bool) int32 {
 	}
 	return 0
 }
+
+// commandPath returns the path that should be used for this command.
+// Per the spec, if it contains a path separator we don't alter it; if not we look it up,
+// honouring the given env var if present.
+func commandPath(command *pb.Command) string {
+	arg := command.Arguments[0]
+	if strings.ContainsRune(arg, filepath.Separator) {
+		return arg
+	}
+	for _, env := range command.EnvironmentVariables {
```

Comment:
We have this in plz too, it's a little tedious. Might put a request in upstream to add something to the stdlib for this, if I can think of a good name for the function.

Reply:
Seems like a reasonable impl.